### PR TITLE
Release v0.27.0

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +12,39 @@
 - Remove `tracetranslator.TagHTTPStatusCode`, use `conventions.AttributeHTTPStatusCode` (#3111)
 - Remove OpenCensus status constants and transformation (#3110)
 - Remove `tracetranslator.AttributeArrayToSlice`, not used in core or contrib (#3109)
+- Remove `internaldata.MetricsData`, same APIs as for traces (#3156)
+- Rename `exporterImp` struct to `exporter` in `otlp` and `otlphttp` exporters (3160)
```
Comment (bogdandrutu) [inline CHANGELOG.md:16]: This is private, no breaking change

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,12 +12,39 @@
...
+- Rename `config.IDFromString` to `NewIDFromString`, remove `MustIDFromString` (#3177)
+- Mark internaldata traces translation as deprecated for external usage (#3176)
```
Comment (bogdandrutu) [inline CHANGELOG.md:18]: This can be "enhancements"

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,12 +12,38 @@
...
+- Remove locking from jaeger receiver start and stop processes (#3070)
+- Fix batch processor metrics reorder, improve performance (#3034)
+- Fix batch processor traces reorder, improve performance (#3107)
+- Fix batch processor logs reorder, improve performance (#3125)
+- Avoid one unnecessary allocation in grpc OTLP exporter (#3122)
+- Validate that batch config max size is greater than send size (#3126)
+- Add capabilities to consumer, remove from processor (#2770)
+- Remove internal protos usage in Prometheusremotewrite exporter (#3184)
+- `prometheus` receiver: Honor Prometheus external labels (#3127)
+- Canonicalize enum names in pdata. Fix usage of uppercase names (#3208)
```
Comment (bogdandrutu) [inline CHANGELOG.md:45]: This is a breaking change.

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,12 +12,38 @@
...
 
 - Fix Prometheus receiver metric start time and reset determination logic. (#3047)
   - The receiver will no longer drop the first sample for `counter`, `summary`, and `histogram` metrics.
 - The Prometheus remote write exporter will no longer force `counter` metrics to have a `_total` suffix. (#2993)
+- Remove locking from jaeger receiver start and stop processes (#3070)
+- Fix batch processor metrics reorder, improve performance (#3034)
+- Fix batch processor traces reorder, improve performance (#3107)
+- Fix batch processor logs reorder, improve performance (#3125)
+- Avoid one unnecessary allocation in grpc OTLP exporter (#3122)
+- Validate that batch config max size is greater than send size (#3126)
```
Comment (bogdandrutu) [inline CHANGELOG.md:41]: - Validate that batch config max size is greater than send size (#3126)

=>

- Batch processor: Validate that batch config max size is greater than send size (#3126)

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**